#include "record.h"
#include "cmpt_error.h"
#include <iostream>
#include <vector>

using namespace std;

    record::record(){
        cout << "It worked!\n";
    }

    record::record(const record& other){

    }

    record::~record(){

    }

    int record::get_name() const{

    }

    int record::get_time() const{

    }

    int record::get_meal() const{

    }

    int record::get_ingreds() const{

    }

    int record::get_diet() const{

    }

    int record::get_ingred(int i) const{

    }

    int record::get_ingred_item(int i) const{

    }

    int record::get_ingred_amount(int i) const{

    }

    int record::set_name() const{

    }

    int record::set_time() const{

    }

    int record::set_meal() const{

    }

    int record::set_ingreds() const{

    }

    int record::set_diet() const{

    }

    int record::set_ingred(int i) const{

    }

    int record::set_ingred_item(int i) const{

    }

    int record::set_ingred_amount(int i) const{

    }

    void record::print() const{

    }
